requirements-dev.txt

```diff
@@ -1,11 +1,11 @@
-alembic>=1.13
```

Comment:
Could be a different PR, maybe next time.

Reply:
Shall separate next time - I had to add APScheduler to the list, so I thought I might as well update them while I was editing the files